.travis.yml

```diff
@@ -1,15 +1,32 @@
 language: c
-sudo: required
-dist: trusty
-branches:
-  only: master
+os: linux
+dist: xenial
 install:
-  - wget https://raw.githubusercontent.com/ocaml/ocaml-travisci-skeleton/master/.travis-opam.sh
-  - wget https://raw.githubusercontent.com/simonjbeaumont/ocaml-travisci-skeleton/docgen/.travis-docgen.sh
-  - wget https://raw.githubusercontent.com/simonjbeaumont/ocaml-travis-coveralls/master/travis-coveralls.sh
-script: bash -ex .travis-opam.sh && bash -ex .travis-docgen.sh && bash -ex travis-coveralls.sh
+  - wget https://raw.githubusercontent.com/ocaml/ocaml-ci-scripts/master/.travis-opam.sh
+script: bash -ex .travis-opam.sh
+jobs:
+  include:
```

Comment:
Is this the list of builds run on Travis? It seems a bit excessive. Maybe limit this to 4.07 to 4.10?

Reply:
All Ocaml versions are supported upstream, starting from 4.02. There is even one fix you backported so the code would work in versions prior to 4.05.